template.yaml

```diff
@@ -87,6 +93,7 @@ spec:
         repoVisibility: public
         topics:
           - ${{ parameters.product }}
+          - ${{ parameters.jenkins_topic }}
```

Comment:
what do you think about just harding it to the cft one for labs? then we wouldn't need to add to the UI

Reply:
@zc-hmcts ?

Reply:
with a default value it should take away the hassle element of it, but otherwise if adding a topic is going to be something they need to think about when they're creating a repo "irl" then it seems like a good thing to have here